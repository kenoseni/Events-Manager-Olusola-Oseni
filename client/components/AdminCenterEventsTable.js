import React, {Component} from 'react';

class AdminCenterEventsTable extends Component {
  constructor(props) {
    super(props);
  }

  render () {
    const {event, i} = this.props
    return (
      <tr>
        <th scope="row">{i}</th> 
        <td>{event.name.toUpperCase()}</td>
        <td>{event.date}</td>
        <td>{event.time}</td>
      </tr>
    )
  }
}
export default AdminCenterEventsTable;
