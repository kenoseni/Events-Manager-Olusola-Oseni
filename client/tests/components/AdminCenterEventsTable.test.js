import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import AdminCenterEventsTable from '../../components/AdminCenterEventsTable';

configure({ adapter: new Adapter() });

let props = {
  event: {
    name: 'Python quiz',
    startDate: '2018-12-04',
    endDate: '2018-12-05',
    time: '4 pm'
  },
  i: 1
}

const wrapper = shallow(<AdminCenterEventsTable
  {...props}
/>)

describe('<AdminCenterEventsTable />', () => {
  it('should render a <AdminCenterEventsTable /> component', () => {
    expect(wrapper).toMatchSnapshot();
  });
})
