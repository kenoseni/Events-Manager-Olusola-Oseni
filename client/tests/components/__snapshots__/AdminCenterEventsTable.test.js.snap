// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminCenterEventsTable /> should render a <AdminCenterEventsTable /> component 1`] = `
<tr>
  <th
    scope="row"
  >
    1
  </th>
  <td>
    PYTHON QUIZ
  </td>
  <td>
    2018-12-04
  </td>
  <td>
    2018-12-05
  </td>
  <td>
    4 pm
  </td>
</tr>
`;
